### ISSUE TITLE
Fixture timeout exceeded with Playwright test runner

### ISSUE THREAD

**Issue by becem-gharbi** [nuxt/test-utils#861]
### Environment

- Operating System: Linux
- Node Version:     v18.20.3
- Nuxt Version:     3.11.2
- CLI Version:      3.11.1
- Nitro Version:    -
- Package Manager:  pnpm@8.15.6
- Builder:          -
- User Config:      devtools
- Runtime Modules:  -
- Build Modules:    -

### Reproduction

https://stackblitz.com/edit/github-jpkifv?file=playwright.config.ts

### Describe the bug

Hi,
When running a fixture test with Playwright, the setup performing the build step takes too long which leads to a timeout error.

### Additional context

The reason seems to be related to the `setupTimeout` option not passed as described in Playwright's docs https://playwright.dev/docs/next/test-fixtures#fixture-timeout.

https://github.com/nuxt/test-utils/blob/66b23ae6f3d717690551ff1908647ff0f0cfa63b/src/playwright.ts#L40

### Logs

```shell-script
Fixture "_nuxtHooks" timeout of 30000ms exceeded during setup.
```

**Comment by josip2312** [comment]
👍 

I don't get why the build step is ran for every test case. In a bigger app it takes 30 seconds for each test to finish setup.

**Comment by becem-gharbi** [comment]
> I don't get why the build step is ran for every test case. In a bigger app it takes 30 seconds for each test to finish setup.

I agree. As a solution, I suggest launching your Nuxt app with Playwright. https://playwright.dev/docs/test-webserver#configuring-a-web-server

**Comment by josip2312** [comment]
> > I don't get why the build step is ran for every test case. In a bigger app it takes 30 seconds for each test to finish setup.
> 
> I agree. As a solution, I suggest launching your Nuxt app with Playwright. https://playwright.dev/docs/test-webserver#configuring-a-web-server

Does not seem to work, I used this as a workaround for now https://github.com/nuxt/test-utils/issues/314#issuecomment-2047815314

**Comment by yamachi4416** [comment]
Thank you for the report.
This has been fixed in the latest version, so closing this issue.

Reproduction:
- repo: https://github.com/yamachi4416/reproductions/tree/main/nuxt-test-utils/issues-861
- result: https://github.com/yamachi4416/reproductions/actions/runs/34750104936/job/103704993200
